ksql-common/src/main/java/io/confluent/ksql/util/SchemaUtil.java

```diff
@@ -363,4 +375,40 @@ public static Schema ensureOptional(final Schema schema) {
         .build();
   }
 
+
+  public static boolean compareSchemaTypes(final Schema arg1, final Schema arg2) {
```

Comment:
since this is now a public API, we should create dedicated tests for it in `SchemaUtil`